blocksuite/framework/store/src/transformer/middleware.ts

```diff
@@ -35,6 +36,8 @@ export type BeforeExportPayload =
   | {
       model: DraftModel;
       type: 'block';
+      action: 'default' | 'skip';
```

Comment:
What's the meaning of this new action and transformer?

Reply:
I want to skip some models during export to snapshot without modifying the draf model.

Reply:
Understood, but why you don't want to modify the draft model? I think `model.children.filter` is more straight forward than adding flags.

Reply:
Oh I see. I thought the `DraftModel` is just an other name of `BlockModel`. So we can modify the draft model without affecting the original `BlockModel` ?

Reply:
It seems like `toDraftModel` has not been called.